packages/firestore/src/local/simple_db.ts

```diff
@@ -322,6 +322,17 @@ export class SimpleDb {
                   'data for your app to start fresh.'
               )
             );
+          } else if (error.name === 'InvalidStateError') {
+            reject(
+              new FirestoreError(
+                Code.FAILED_PRECONDITION,
+                'Unable to open an IndexedDb connection. This could be due to running in a ' +
+                  'private browsing session on a browser whose private browsing sessions do not ' +
+                  'support IndexedDb. (' +
```

Comment:
Nit: I think you should move the period after the parenthesis or use `:` without parenthesis.

Reply:
Done (changed to ":").